transmission_interface/package.xml

```diff
@@ -17,17 +17,18 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <depend>tinyxml</depend>
-  <depend>roscpp</depend>
   <depend>pluginlib</depend>
-  <build_depend>hardware_interface</build_depend>
+  <depend>roscpp</depend>
+  <depend>tinyxml</depend>
```

Comment:
Is this a header-only tinyxml version by any chance?

Reply:
I'm not aware of any header-only TinyXML versions. This is just the system-provided TinyXML w/ the ROS `cmake_modules`.

Reply:
Right. I've dug in a bit and found that the `tinyxml` rosdep symbol resolves to `libtinyxml-dev` which also pulls in the library. Not ideal, as one would not want to carry the `-dev` package with them as a `run_depend` but it seems we don't have a choice.

Reply:
Could consider contributing a `rosdep` rule to split the runtime from the devel package?